prow/plugins/lgtm/lgtm.go

```diff
@@ -61,7 +61,7 @@ type commentPruner interface {
 }
 
 func init() {
-	plugins.RegisterGenericCommentHandler(PluginName, handleGenericCommentEvent, helpProvider)
```

Comment:
Plugins are a core Prow thing. I would be extremely surprised if anyone outside of the Prow core were using this interface, and this is not exposed as a reusable library. Why not just refactor the signature? If you want you could plumb down e.g. a `context` to decorate with values.

Reply:
I wanted to avoid full refactor as that will be used most probably in two plugins: `lgtm` and `approve`, but if you insist I can do that. I would keep the status object though, it seems to be more expressive.

Reply:
Although keeping bool in a whole refactored environment will most likely mean losing one state: not used ("") in metrics, so I don't know, probably pointer could be the solution.